app/system-modules/profile/views/details.ejs

```diff
@@ -0,0 +1,22 @@
+<div class="panel panel-default" style="padding: 10px">
+  <h3> <%= user.firstName  %> <small> <%= user.username  %></small></h3> 
```

Comment:
Like we discussed, this should be the user's username in the h3, then the word _username_ in the small.